### DISCUSSION TITLE
电脑端收不到 mirai 发出的消息

### DISCUSSION THREAD

**Discussion by andersgong** [mamoe/mirai#863] [Q&A] [answered]
我的mirai用手机协议登录，收发消息一切正常。
但是用电脑qq登录同一个帐号之后，mirai也能正常收到群消息，并且console显示发出了响应，但是群里收不到响应。然后我把电脑QQ退出，等了10分钟以上，还是一样的现象，console显示一切正常，能收能发，没有报错，群里收不到mirai发出的消息。

**Comment by Him188** [comment, marked as answer]
只是你登录过于频繁了，先离线几分钟再登录，或者发几条好友消息

**Reply by Him188** [reply]
实际上这也是一个问题，但要修复不太简单。实测离线一会就可以解决那就不太重要了

**Reply by andersgong** [reply]
我不太明白，mirai已经在线一个多小时，一直没有断开，也正常工作。这时候，我用电脑登录同一个号，mirai console还是一切正常的，也能看到正常收发的群消息，但是群里不显示mirai发出的消息。这应该不存在频繁登录吧。。。

**Reply by andersgong** [reply]
我换了个号做同样的实验又是正常的，没有这个问题，那可能真的是太频繁导致的，感谢！
